node-graph/gcore/src/raster/color.rs

```diff
@@ -831,6 +860,26 @@ impl Color {
 		Some(Color::from_rgba8_srgb(r, g, b, a))
 	}
 
+	/// Creates a color from a 8-character RGBA hex string (with a # prefix).
+	///
+	/// # Examples
+	/// ```
+	/// use graphene_core::raster::color::Color;
+	/// let color = Color::from_rgba_str_prefixed("#7C67FA61").unwrap();
+	/// ```
+	pub fn from_rgba_str_prefixed(color_str: &str) -> Option<Color> {
+		if color_str.len() != 9 {
+			return None;
+		}
+   		let color_str = &color_str[1..];
+		let r = u8::from_str_radix(&color_str[0..2], 16).ok()?;
+		let g = u8::from_str_radix(&color_str[2..4], 16).ok()?;
+		let b = u8::from_str_radix(&color_str[4..6], 16).ok()?;
+		let a = u8::from_str_radix(&color_str[6..8], 16).ok()?;
+
+		Some(Color::from_rgba8_srgb(r, g, b, a))
+	}
```

Comment:
Please reuse the existing function above to reduce code duplication. e.g. `colour_string.strip_prefix("#").and_then(|value|Color::from_rgba_str(value)`. This will also ensure that it is actually a `#`.